docs/af5-app-from-scratch/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,52 @@
+:navtitle: Introduction
+:reftext: Building an Axon Framework 5 Application from Scratch
+
+= Introduction
+
+Welcome to the Axon Framework 5 Getting Started Guide!
+This tutorial is based on the version `5.0.0-M1`.
+
+Building a new application from scratch can be daunting, especially when learning something new.
+This tutorial will guide you through the various stages of development,
+starting with creating your first project and explaining the capabilities of Axon Framework 5.
+
+The Axon Framework 5 introduces a lot of changes,
+so even if you are already familiar with Axon Framework 4, we still recommend you to go through this tutorial.
+
+== What's New in Axon Framework 5?
+Axon Framework 5 represents a significant evolution with major improvements to the framework's architecture:
+
+* Dynamic Consistency Boundary (DCB) - A revolutionary concept that allows more flexible event sourcing patterns overcoming traditional aggregates limitations. With this mind-shift you may organize your code more around features than entities. It plays nicely with Vertical Slice Architecture.
+* Revisited Configuration - A new, more flexible configuration model.
+* Improved Testing Support - Enhanced testing fixtures that allow you to verify your app configured as same as in the production code.
+* Java 21 Required - Taking advantage of the latest Java features.
+
+Whether you're new to Axon Framework or upgrading from version 4,
+this tutorial will help you understand these new concepts through practical examples.
+
+== About this tutorial
+This tutorial will guide you through creating a "University Registration Application" using Axon Framework 5.
+We'll follow a test-first approach, implementing features using Vertical Slice Architecture.
```

Comment:
```suggestion
We will implement features using Vertical Slice Architecture using Test-Driven Development.
```

Reply:
I prefer to use "Test First" name for that. From Martin Dilger book nad Adam Dmytriuk also say that he do "test first", not TDD, because I don't follow strictly the TDD methodology - the TDD doesn't influence my code/messages - I just specify my scenarios using GWT with messages and do not follow red->green->refactor lifecycle and one test after another. I think it's safer to stick to "Test First" because someone may say it's not TDD here. 

![image](https://github.com/user-attachments/assets/63425f1b-2d5a-4bc2-b421-497170a152c3)